lib/Frontend/ArgsToFrontendOptionsConverter.cpp

```diff
@@ -86,6 +86,9 @@ bool ArgsToFrontendOptionsConverter::convert(
   Opts.RemarkOnRebuildFromModuleInterface |=
     Args.hasArg(OPT_Rmodule_interface_rebuild);
 
+  Opts.ExperimentalCompressedDependencies |=
```

Comment:
Just for my own edification, why |= instead of =?  Is there any other place that might set this to true? The debugger?